docs/src/pages/customization/components/DynamicClassName.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import clsx from 'clsx';
+import { makeStyles } from '@material-ui/core/styles';
+import Button from '@material-ui/core/Button';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+import Switch from '@material-ui/core/Switch';
+
+const useStyles = makeStyles({
+  button: {
```

Comment:
I would approve but, I can’t remember if we include createStyles still. I would guess we still do but I’d have to check to be 100% 

I’m on my phone atm but will check in a bit

Reply:
I think that we add it when the theme is used, it seems to be fine here.